### PR TITLE
Update reference.md

### LINKED ISSUE #1724: Reporting bugs section in docs references to Launchpad
Seems like Elementary uses GitHub for development for a while but [this section](https://elementary.io/docs/code/reference#reporting-bugs) in docs refers to Launchpad which is confusing for new users (like me)

### PR DESCRIPTION
**Pull request by danrabbit**
This updates the bug reporting guide for GitHub

Fixes #1724 

It doesn't attempt to touch the section about suggesting design changes

### CODE REVIEW

File: docs/code/reference.md [hunk 1]
```diff
@@ -218,20 +217,17 @@ First of all, it's important to mention that bug reports should be written in En
...
-If a report affects you, mark it using the link. Do not confrim your own reports. Even if you are 100% sure that you are right, just don't do it. It's not good bug tracker etiquette. A bug manager or developer for the app will confirm the report when they are able to review it.
-
 ## Don't Make "Me Too" Comments {#dont-make-me-too-comments}
-We mentioned this earlier, but it's worth mentioning again. Do not make comments that simply say "This issue affects me as well". This clutters up the tracker. Only comment if you are providing additional information that helps find the source of the issue. If you only want to let a developer know that you are affected, use the green link that says, "Does this bug affect you?" under the report's title.
+We mentioned this earlier, but it's worth mentioning again. Do not make comments that simply say "This issue affects me as well". This clutters up the tracker and can make it difficult to find important information that may solve the issue. Only comment if you are providing additional information that helps find the source of the issue. If you only want to let a developer know that you are affected, use the reactions system to add a 👍.
```
Comment (lewisgoddard) [inline docs/code/reference.md]: Does this thumbs up show correctly? Would we be better off adding words?

Comment (danrabbit) [inline docs/code/reference.md]: It should, are you not seeing it?

Comment (lewisgoddard) [inline docs/code/reference.md]: It just doesn't look great, and I doubt that unicode icon is well supported. image

**Comment by cassidyjames** [comment]
Probably safer to use Font Awesome instead of relying on client emoji support

File: docs/code/reference.md [hunk 2]
```diff
@@ -218,20 +217,17 @@ First of all, it's important to mention that bug reports should be written in En
...
-
 ## Don't Make "Me Too" Comments {#dont-make-me-too-comments}
-We mentioned this earlier, but it's worth mentioning again. Do not make comments that simply say "This issue affects me as well". This clutters up the tracker. Only comment if you are providing additional information that helps find the source of the issue. If you only want to let a developer know that you are affected, use the green link that says, "Does this bug affect you?" under the report's title.
+We mentioned this earlier, but it's worth mentioning again. Do not make comments that simply say "This issue affects me as well". This clutters up the tracker and can make it difficult to find important information that may solve the issue. Only comment if you are providing additional information that helps find the source of the issue. If you only want to let a developer know that you are affected, use the reactions system to add a <i class="fa fa-thumbs-o-up" aria-hidden="true" title="thumbs up"></i>.
```
Comment (lewisgoddard) [inline docs/code/reference.md]: I'm not sure we want the `aria-hidden="true"` on these, they don't make much sense without accessibility including them.

**Comment by danrabbit** [comment]
You got it :+1:

**Pull request merged**